README.md

```diff
@@ -12,7 +12,54 @@ Kyma CLI requires the following software:
 
 ## Installation
 
-For the installation instructions, see the [release page](https://github.com/kyma-project/cli/releases).
+Following commands will install Kyma CLI from the latest release. To install another release, set `KYMA_VERSION` to the desired version in the first command.
+
+### Installation on Linux (i386)
```

Comment:
```suggestion
### Linux (i386)
```
I would just go with `Linux`, `Windows` and so on as it is already known that it's an installation guide.